Sources/StreamChatUI/ChatMessageList/ChatMessageListCollectionView.swift

```diff
@@ -306,3 +306,252 @@ open class ChatMessageListCollectionView<ExtraData: ExtraDataTypes>: UICollectio
         return indexPathsForVisibleItems.contains(lastIndexPath)
     }
 }
+
+// ======
```

Comment:
You left a lot of `collectionView` in the comments on several methods.

Maybe remove the collection view code and rename the file? or create a separate file for the table view.

Reply:
✅ Done